editor/tests/org.fusesource.ide.projecttemplates.tests.integration/src/main/java/org/fusesource/ide/projecttemplates/wizards/pages/FuseIntegrationWizardRuntimeAndCamelPageIT.java

```diff
@@ -19,12 +19,14 @@
 import org.fusesource.ide.projecttemplates.wizards.FuseIntegrationProjectWizard;
 import org.junit.After;
 import org.junit.Before;
+import org.junit.Ignore;
 import org.junit.Test;
 
 /**
  * @author lhein
  *
  */
+@Ignore("This test no longer applies as user can select whatever version they want as long as it exists in a public repo defined. Maybe add a test if the check for availability works!")
```

Comment:
I guess check for availability is implemented now, so would be nice to add the test in this PR or to create a subtask in JIRA for it